### PR TITLE
[WIP] Add flatpages and ckeditor for a light cms experience

### LINKED ISSUE #440: Help centre with articles that can be written in the admin
It'd be nice to start transitioning pages from google docs into the website but I'd prefer not to always be on-call for them. An approach here could be to use something like flatpages to allow the content to be updated through the admin (maybe with a wysiwyg editor, markdown, etc) and then render it from DB. That way I wouldn't have to take requests to manually update the content all the time

### PR DESCRIPTION
**Pull request by samihibrahim**
- Closes #440.
- Still looking into ways of making the editing experience easier (e.g. add defaults, remove extra fields, ...)

### CODE REVIEW

File: website/public/admin.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
-# from django.contrib import admin
+from django.contrib import admin
```
Comment (samihibrahim) [inline website/public/admin.py:1]: Is this a good place for this or should I have used another `admin.py` file? 🤔

Comment (ryanwilsonperkin) [reply website/public/admin.py:1]: Good question, we're using the `public` app as a source of things that should be accessible to the public without logging into the website so FlatPages-related stuff seems like it should fit well here (the "admin" stuff is never accessible to the public anyways)

File: website/templates/flatpages/default.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
```
Comment (ryanwilsonperkin) [inline website/templates/flatpages/default.html:1]: We can also tweak the `base.html` template if you need to include other sections from flatpages (not sure if there's more needed beyond the title/content)

**Review by ryanwilsonperkin — COMMENTED** [review]
Nice work :) Let me know if there's any help I can offer with this issue

**Comment by ryanwilsonperkin** [comment]
Closing as stale, please let me know if you're still interested in taking on this work

**Pull request closed without merging**